applications/FluidDynamicsApplication/tests/cpp_tests/test_two_fluid_navier_stokes_element.cpp

```diff
@@ -240,6 +240,132 @@ namespace Kratos {
+				for(unsigned int k=0; k<3; k++){
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY)[k]    = vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 1)[k] = 0.9*vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(VELOCITY, 2)[k] = 0.75*vel_original(i,k);
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY)[k]    = 0.0;
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 1)[k] = 0.0;
+					pElement->GetGeometry()[i].FastGetSolutionStepValue(MESH_VELOCITY, 2)[k] = 0.0;
+				}
+			}
+
+			pElement->GetGeometry()[0].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[1].FastGetSolutionStepValue(DISTANCE) =  1.0;
+			pElement->GetGeometry()[2].FastGetSolutionStepValue(DISTANCE) = -1.0;
+			pElement->GetGeometry()[3].FastGetSolutionStepValue(DISTANCE) =  1.0;
+
+			pElement->GetGeometry()[0].SetValue(CURVATURE, 1.0);
+			pElement->GetGeometry()[1].SetValue(CURVATURE, 1.0);
+			pElement->GetGeometry()[2].SetValue(CURVATURE, 1.0);
+			pElement->GetGeometry()[3].SetValue(CURVATURE, 1.0);
+
+			Vector inner_pressure_grad = ZeroVector(3);
+			Vector outer_pressure_grad = ZeroVector(3);
+
+			for(unsigned int k=0; k<3; k++){
+				inner_pressure_grad[k] = 0.5*k;
+				outer_pressure_grad[k] = 1.5*k + 1.0;
+			}
+
+			pElement->GetGeometry()[0].SetValue(PRESSURE_GRADIENT, inner_pressure_grad);
+			pElement->GetGeometry()[1].SetValue(PRESSURE_GRADIENT, outer_pressure_grad);
+			pElement->GetGeometry()[2].SetValue(PRESSURE_GRADIENT, inner_pressure_grad);
+			pElement->GetGeometry()[3].SetValue(PRESSURE_GRADIENT, outer_pressure_grad);
+
+			// Compute RHS and LHS
+			Vector RHS = ZeroVector(16);
+			Matrix LHS = ZeroMatrix(16,16);
+
+			const auto& r_process_info = modelPart.GetProcessInfo();
+			pElement->Initialize(r_process_info); // Initialize the element to initialize the constitutive law
+			pElement->CalculateLocalSystem(LHS, RHS, r_process_info);
+
+			// Check the RHS values (the RHS is computed as the LHS x previous_solution,
+			// hence, it is assumed that if the RHS is correct, the LHS is correct as well)
+			KRATOS_CHECK_NEAR(RHS(0), -268.0263579, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(1), -4.183181586, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(2), -81.36338392, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(3), -0.07197510975, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(4), 56.06058861, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(5), 29.91012642, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(6), -428.7793345, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(7), 0.3701129402, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(8), 107.982446, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(9), 26.13457374, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(10), -466.3218786, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(11), 0.02197510976, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(12), 149.4624899, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(13), 43.2426481, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(14), -514.3062363, 1e-7);
+			KRATOS_CHECK_NEAR(RHS(15), -0.4201129402, 1e-7);

```

Comment:
Use the `KRATOS_CHECK_VECTOR_NEAR` macro in here.

Reply:
Done.